torch/csrc/jit/script/function_schema_parser.cpp

```diff
@@ -185,11 +195,11 @@ struct SchemaParser {
       std::vector<IValue> vs) {
     switch (kind) {
       case TypeKind::FloatType:
-        return fmap(vs, [](IValue v) { return v.toDouble(); });
+        return fmap_<double>(vs, [](IValue v) { return v.toDouble(); });
```

Comment:
I really don't see why we can't have constructors that convert std::vector into IValues. Not having them is adding all of this extra code.